cypress/e2e/admin/administrateUsers.feature

```diff
@@ -7,6 +7,7 @@ Feature: Admin - To add, edit and delete new users by the admin.
 
     Scenario: Adding a new user, edit and delete this user
```

Comment:
I think we can mention it in one sentence instead of two separate sentences

Reply:
ok, but this is not changed in this PR, its like this before, but I can change.